10/umbraco-deploy/deploy-settings.md

```diff
@@ -152,9 +156,12 @@ All of these times are configured using [standard timespan format strings](https
 
 Even with appropriate settings of the above timeouts, Deploy's backoffice transfer operations can hit a hard limit imposed by the hosting environment. For Azure, this is around 4 minutes. This will typically only be reached if deploying a considerable amount of items in one go. For example, a media folder with thousands of items can reach this limit.
 
-An error message of `500 - The request timed out. The web server failed to respond within the specified time.` will be reported.
+An error message of "The remote API has returned a response indicating a platform timeout" will be reported.
 
-If encountering this issue, the `SourceDeployBatchSize` setting can be applied with an integer value (for example 1000). This will cause Deploy to transfer items in batches, up to a maximum size. This will allow each individual batch to complete within the time available.
+If encountering this issue, there are two batch settings that can be applied with integer values (for example 500). This will cause Deploy to transfer items in batches, up to a maximum size. This will allow each individual batch to complete within the time available.
+
+- `SourceDeployBatchSize` - applies a batch setting for the transfer of multiple selected items to an upstream environment (such as a media folder with many images).
```

Comment:
This is the note as will be displayed on the Deploy dashboard:

> Platform limits can be reached when transferring large amounts of content or media. To try to avoid this, batch sizes for deploy operations can be configured.
> 
> When set, this setting controls how many items are selected per batch to transfer to an upstream environment, potentially resulting in multiple deployments.
> 
> This will take effect only for transfers to upstream environments (not for restores) and when multiple items are queued for transfer, such as a single media folder containing many files (queued using 'Include all items below'). Each batch will still include all determined dependencies and relations to successfully complete the deployment.